main.c

```diff
@@ -0,0 +1,30 @@
+#include <stdio.h>
+
+void revrse(char *pointer)
+{
+  int len = 0,i = 0;

```

Comment:
int len = 0,i = 0;
Better to separate on two lines

Reply:
Done